packages/providers/audit-logs-local/src/models/audit-logs.ts

```diff
@@ -0,0 +1,32 @@
+import type { Model } from '@strapi/database';
+
+const auditLog: Model = {
+  uid: 'admin::audit-log',
+  tableName: 'strapi_audit_logs',
+  singularName: 'audit-log',
+  attributes: {
+    id: {
+      type: 'increments',
+    },
+    action: {
+      type: 'string',
+      required: true,
+    },
+    date: {
+      type: 'datetime',
+      required: true,
+    },
+    // @ts-expect-error database model is not yet updated to support useJoinTable
+    user: {
```

Comment:
if this schema is different from v4, we need to create a migration to keep v4 data. I believe all audit-log data will be transfered except the users in this situation.